code/hud/hudparse.cpp

```diff
@@ -391,6 +394,8 @@ void parse_hud_gauges_tbl(const char *filename)
 				}
 				break;
 			case 1:
+				mprintf(("$Ships in hud_gauges.tbl and -hdg.tbms is deprecated. Use \"$Name:\" and define Gauge Config Settings in ships.tbl per-ship instead.\n"));
```

Comment:
Is this truly getting deprecated?  This sounds more like a QOL upgrade.

Reply:
Not really but when $ships replaced $ship for convenience a similar warning was added so I just followed the pattern. 🤷‍♂️

Reply:
Hmmmm ... I wish I had noticed that and said something. I don't want modders to think that the old method is disallowed if it's not....  

Maybe?

`$Ships in hud_gauges.tbl and -hdg.tbms can be replaced. Gauge Config Settings can now be defined per-ship in ships.tbl. \n`

I'm having trouble wording "disfavored" but not "disallowed" without sounding like I'm from 1850.

Reply:
Good enough for government work. Changed.